src/imagej/convert.py

```diff
@@ -474,9 +474,9 @@ def index_img_to_roi_tree(
     :param index_img: An ImgLib2 or NumPy index image
     :return: A ROITree with ImgLib2 ROIs
```

Comment:
Would it not make more sense for this logic to exist as a `Converter` plugin in imagej-common, and then simply call it here in Python?

Reply:
Totally agree here! I made this https://github.com/imagej/imagej-common/issues/111 issue about adding the index image --> ROI functions as a converter plugin.